### PR TITLE
Migrating legacy guides to a new site

### PR DESCRIPTION
**Pull request by Wulingoh**
All legacy guides have been migrated and updated with social media tags except for the payment flow and merchant terminal guides. 

Test plans: 
1. Visit https://docs.centrapay.com/guides/creating-test-money

Screen size for 1980px and over:
2. The content in the guide is positioned at the center of the page for desktop
3. The guide titles appear on the sidebar on the left. 
4. The links in each guide are working and not broken. 
5. The guides should appear on the new site
6. Only Payment flow and merchant terminal guides remained on the old site
eg: 
The new site:
Screenshot 2023-02-15 at 9 16 49 AM

The old site:
Screenshot 2023-02-15 at 9 16 22 AM

7. Screen over 4k:
Screenshot 2023-02-15 at 9 15 16 AM

8. eg:
Screenshot 2023-02-09 at 12 48 13 PM

Mobile:
9. The guide titles appear after clicking the menu button.
eg: 
Screenshot 2023-02-15 at 9 38 07 AM

10. The content is positioned below the navigation bar and centered: 
eg:
Screenshot 2023-02-09 at 2 12 51 PM

### CODE REVIEW

File: content/guides/creating-test-money.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: Creating Test Money
+description: Use test assets to make testing integrations easier.
+nav:
+  path: Reference/Digital Assets
```
Comment (Ray2126) [inline content/guides/creating-test-money.md:5]: Do we need a landing page for this new section (Digital Assets), same for App Integrations? Currently, clicking on the section header goes no where.

File: content/guides/ecommerce-website.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+nav:
+  path: Reference/Merchant Integrations
+  order: 12
+---
+
+eCommerce websites accepting payments via Centrapay may want to redirect their users back to their website after they pay/cancel the payment request. In order to do this, the eCommerce website must add `allowedRedirectUrls` to it’s [Merchant Config](https://docs.centrapay.com/api/merchant-configs).
+
+After `allowedRedirectUrls` have been added to the Merchant Config, the eCommerce website can simply pass in their `redirectUrl` when creating a [Payment Request](https://docs.centrapay.com/api/payment-requests).
+
+POST `/api/payment-requests`
```
Comment (Ray2126) [inline content/guides/ecommerce-website.md:13]: This section looks really strange on the website.

Comment (Ray2126) [reply content/guides/ecommerce-website.md:13]: image

File: content/guides/creating-test-money.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+title: Creating Test Money
```
Comment (Ray2126) [inline content/guides/creating-test-money.md:2]: We should delete these pages from the legacy guides as well :)

**Review by Ray2126 — CHANGES_REQUESTED** [review]
FYI the PR title seems incorrect, it should say something about moving legacy guides over to the new site.

File: content/guides/creating-test-money.md (hunk 1 above)
Comment (Wulingoh) [reply content/guides/creating-test-money.md:5]: I have amended the changes and clicking the section header will land on the first guide of the doc.

File: content/guides/ecommerce-website.md (hunk 2 above)
Comment (Wulingoh) [reply content/guides/ecommerce-website.md:13]: Could you please elaborate on this? Thanks. 🙏

File: content/guides/creating-test-money.md (hunk 3 above)
Comment (Wulingoh) [reply content/guides/creating-test-money.md:2]: I have removed the pages from the legacy guides.

File: content/guides/ecommerce-website.md (hunk 2 above)
Comment (Wulingoh) [reply content/guides/ecommerce-website.md]: Megan came and helped to elaborate :)
Screenshot 2023-02-10 at 11 33 04 AM

File: content/guides/payment-terminals.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+---
```
Comment (Ray2126) [inline content/guides/payment-terminals.md:1]: There's a comment in the [Migrating remaining legacy guides](https://www.notion.so/centrapay/Migrating-remaining-legacy-guides-a15efbe41db04386937ea16862264220) Notion meeting notes page about redirecting to this page on the `/guides/merchant-terminals` path. 

I think we should do something like this: https://medium.com/dont-leave-me-out-in-the-code/nuxt-redirects-in-4-simple-steps-9a0d5cb4c0d

I also think the redirect url should be defined in the front matter on each page.

Let me know if you need a hand with this :)

File: content/guides/payment-terminals.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+Centrapay QR codes should be displayed with a Centrapay logo in the centre.
+
+## Integration Architecture
+
+We strongly recommend Centrapay APIs are invoked from your backend and not directly from your payment terminals. Centralizing the invocation of our APIs from your backend offers the following benefits:
+
+1. Merchant network administrators will not need to make additional firewall changes.
+2. Your Centrapay API Keys can be managed centrally inside your secured network.
+3. TLS negotiation with Centrapay APIs can be managed independently of terminal hardware and software updates.
```
Comment (Ray2126) [inline content/guides/payment-terminals.md:60]: In the [Notion guide](https://www.notion.so/centrapay/Payment-Terminal-Guide-fa8dc4919b7d408094a7109ec87a9217), there's a section about mitigating network issues, and that is should be replaced with a link to the merchant integration error handling page. Let's add that in here :)

File: utils/Navigation.js [hunk 6]
```diff
@@ -38,11 +38,17 @@ class Navigation {
       .sort((a, b) => a.order - b.order);
     pages.forEach(page => navigation.insertPage({ page }));
 
-    // FIXME remove this block once merchant integrations has a landing page
-    const merchantIntegrations = navigation.findItem({ title: 'Merchant Integrations' });
-    if(merchantIntegrations) {
-      merchantIntegrations.to = merchantIntegrations.children[0].to;
-    }
+    const sectionHeaderTitle = [
```
Comment (Ray2126) [inline utils/Navigation.js:41]: Let's put the FIXME back in here. The intention is to have landing pages for all the sections so this block of code won't be needed once that's implemented.

File: content/guides/creating-test-money.md (hunk 1 above)
Comment (Ray2126) [reply content/guides/creating-test-money.md:5]: Ok I think that's good enough for now. Nice one!

File: content/guides/creating-test-money.md (hunk 3 above)
Comment (Ray2126) [reply content/guides/creating-test-money.md:2]: Nice!

File: content/guides/ecommerce-website.md (hunk 2 above)
Comment (Ray2126) [reply content/guides/ecommerce-website.md]: Nice! Looks good!

File: content/guides/payment-terminals.md (hunk 4 above)
Comment (Wulingoh) [reply content/guides/payment-terminals.md:1]: I think this task should be in a follow up PR. ;)

File: content/guides/payment-terminals.md (hunk 5 above)
Comment (Wulingoh) [reply content/guides/payment-terminals.md:60]: Screenshot 2023-02-10 at 4 21 07 PM

File: utils/Navigation.js (hunk 6 above)
Comment (Wulingoh) [reply utils/Navigation.js:42]: Done

File: content/guides/payment-terminals.md (hunk 4 above)
Comment (Ray2126) [reply content/guides/payment-terminals.md:1]: If it's a seperate PR then the `guides/merchant-terminals` path will be broken. That path existed on the legacy guides. We don't want to break any existing links :)

File: content/guides/payment-terminals.md (hunk 5 above)
Comment (Ray2126) [reply content/guides/payment-terminals.md:60]: A small introductory paragraph would be nice :)

File: utils/Navigation.js (hunk 6 above)
Comment (Ray2126) [reply utils/Navigation.js:42]: It isn't just merchant integrations now though is it? This handles the landing pages for all 3 sections. We should update the comment

File: content/guides/payment-terminals.md (hunk 4 above)
Comment (tom-blake) [reply content/guides/payment-terminals.md:1]: @Wulingoh Yes PRs should be small but they should encapsulate a single functional change end-to-end 👍 

As Raymond mentioned - not breaking links for existing users is part of our requirements here. 

I'd suggest a separate PR containing all the work required to migrate the `merchant-terminals` guide (my preferred option would have been small, separate PRs for **each** guide)

**Review by tom-blake — CHANGES_REQUESTED** [review]
Awaiting redirects for old URLs

File: content/guides/payment-terminals.md (hunk 4 above)
Comment (Wulingoh) [reply content/guides/payment-terminals.md]: As per discussion with @tom-blake and @Ray2126, I have reverted the changes for `merchant-terminals.md` and will remain in the legacy folder. Meanwhile, I have deleted the `payment-terminals.md` from Nuxt content guides folder.

File: content/guides/payment-terminals.md (hunk 5 above)
Comment (Wulingoh) [reply content/guides/payment-terminals.md]: Please refer to the conversation above. Thanks

File: utils/Navigation.js (hunk 6 above)
Comment (Wulingoh) [reply utils/Navigation.js:42]: I have updated the changes. :)

File: content/guides/payment-terminals.md (hunk 4 above)
Comment (Ray2126) [reply content/guides/payment-terminals.md]: You should update the description :)

**Comment by Ray2126** [comment]
You should delete the images you have added in the `/public` folder as we are no longer moving the merchant terminals page in this PR.

**Review by Ray2126 — APPROVED** [review]

**Comment by tom-blake** [comment]
@Wulingoh I think your test plan should 1) Check that these guides appear in the new docs site AND 2) no longer appear in the legacy docs site

**Review by tom-blake — APPROVED** [review]

**Pull request merged**